pkg/types/azure/platform.go

```diff
@@ -9,7 +9,7 @@ import (
 var aro bool
 
 // OutboundType is a strategy for how egress from cluster is achieved.
-// +kubebuilder:validation:Enum="";Loadbalancer;NatGateway;UserDefinedRouting
+// +kubebuilder:validation:Enum="";Loadbalancer;NatGateway;NATGatewaySingleZone;UserDefinedRouting
```

Comment:
We should probably remove the `NatGateway` option as well? I am fine doing that in a different PR but I didn't see it in #9773 either, so I think it makes the most sense in this one.